### DISCUSSION TITLE
Blueprint Sends Notification to Phone, but Events empty?

### DISCUSSION THREAD

**Discussion by BoostN** [blakeblackshear/frigate#3484] [Ask A Question]
I've noticed I'll get a "person in my driveway" alert to my HA iOS device.. when I look at the camera, there is in fact a person in the driveway but it's in a masked area that should not fire the event. But, when I look at "events" in the frigate UI I see nothing for that. Are these separate in a sense that a event does not correlate to the ios notification blueprint?

**Comment by NickM-27** [comment]
They are different, the notifications are pulled straight from the MQTT events topic while the events have their own requirements. 

That being said, if an object is masked out it shouldn't show up in the notification. Sounds like you have required_zones for events and just don't have that part set for the notification blueprint?
